site/app/controllers/GlobalController.php

```diff
@@ -116,6 +116,16 @@ public function header() {
                 }
             }
 
+            if ($this->core->getConfig()->isPollsEnabled()) {
+                $sidebar_buttons[] = new Button($this->core, [
+                    "href" => $this->core->buildCourseUrl(['polls']),
+                    "title" => "Polls",
+                    "class" => "nav-row",
+                    "id" => "nav-sidebar-polls",
+                    "icon" => "fa-question-circle"
```

Comment:
Personally I think the poll icon from FA would be better https://fontawesome.com/icons/poll?style=solid
But this is just my opinion so feel free to ignore.